### ISSUE TITLE
Is Degree Important ?

### ISSUE THREAD

**Issue by akgupta0777** [tanaypratap/ama#84]
I am a student who is going to get a college this year...But confused between B.Tech Degree or BCA degree.
I have a huge intrest in coding which I think teaches in BCA very well.
But for a Job everyone suggest a B.Tech Degree.
And see my motive is not to get a JOB 
Just wanted to create something by my code for people...Just like Google created search engine which changed our lives rapidly...But if I failed to create something
Then the job is the only option and i need a  Degree .I hope you understand what i trying to say.

**Comment by tanaypratap** [comment]
I'll answer this in a podcast soon.

You can subscribe to bit.ly/thinkingwithtanay to know when it comes.

**Comment by akgupta0777** [comment]
Thanks @tanaypratap for taking my question for podcast 
I joined a B.Tech College and i am learning coding and skills by self learning and developing apps and games.
I choosed B.Tech only for the degree.

I want you to add something more in your podcast is that 
Suppose an idea came to my mind that is revolutionary.So, Tell us how to execute that idea into real. Researching on the internet gives mixed thoughts some says learn ML ,Some says learn AI and i trapped in that loop. And that Idea never become real
